pkg/api/v1beta3/types.go

```diff
@@ -866,6 +866,53 @@ type PodTemplateList struct {
 	Items []PodTemplate `json:"items" description:"list of pod templates"`
 }
 
+// JobSpec is the specification of a job.
+type JobSpec struct {
+	// Completions is the number of desired completions.
+	Completions int `json:"completions" description:"number of times the job has completed successfully (desired or observed, as appropriate)"`
```

Comment:
Please make this field optional (omitempty), default this value to 1, and make it *int.